include/maths/CDataFrameUtils.h

```diff
@@ -55,6 +55,7 @@ class MATHS_EXPORT CDataFrameUtils : private core::CNonInstantiatable {
     using TDoubleVec = std::vector<double>;
     using TDoubleVecVec = std::vector<TDoubleVec>;
     using TSizeVec = std::vector<std::size_t>;
+    using TFloatVec = std::vector<CFloatStorage>;
```

Comment:
We tend to use `TFloatStorageVec` because we never really use `std::vector<float>`. So whilst you're not necessarily wrong I think I'll keep as is for consistency.